src/test/java/com/wikia/webdriver/testsuites/testSuite.xml

```diff
@@ -176,6 +176,7 @@
             <class name="com.wikia.webdriver.testcases.visualeditor.VETemplateTests"/>
             <class name="com.wikia.webdriver.testcases.visualeditor.VisualEditorMultiplePublishTests"/>
             <class name="com.wikia.webdriver.testcases.wampagetests.WamPageTests"/>
+            <class name="com.wikia.webdriver.testcases.flagTests.flagTest"/>
```

Comment:
packages should be write lowercase and test classes should be with first uppercase letter ```com.wikia.webdriver.testcases.flagtests.FlagTest```